modules/agent-host-roles.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/preparing-to-install-with-agent-based-installer.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="agent-host-roles_{context}"]
+= Host roles
+
+Each host in the cluster is assigned a role of either `master` or `worker`.
+You can define the role for each host in the `agent-config.yaml` file by using the `role` parameter.
+If you do not assign a role to the hosts, the roles will be assigned at random during installation.
```

Comment:
Worth noting again here that the host with the rendezvousIP must be assigned the master role, whether by the user or by us automatically. (I'm sure we already mention this in the section about setting the rendezvousIP.)

It does _not_ need to be explicit, but other config cannot conflict. e.g. if you have 4 hosts with 3 explicitly assigned as masters and the other one undefined, then the undefined one can only become a worker and therefore it must not be the rendezvous host.

Reply:
@zaneb I'm not sure I see anything mentioning this as explicitly as you do here, so I'll find a way to elaborate on that in this section.

@andfasano I can try stating that we recommend users explicitly define their host roles. If you want me to provide any reasoning, please let me know and I can include that too.